docs/UserGuide.md

```diff
@@ -173,6 +173,20 @@ Examples:
 * `remark 1 r/Financial Issues` Adds the remark of the 1st person to be `Financial Issues`.
 * `remark 1 r/` Clears remarks (if any) of the 1st person.
 
+### Sorting the person list : `sort`
+
+Sorts the list of persons being viewed by name or date of last visit in ascending or descending order.
+
+Format: `sort paramater/order`
+
+* Sorts the displayed list of persons according to the specified order.
+* Order can be specified as ascending by leaving the order blank or **asc** **ascending**
```

Comment:
Maybe a "/" between **asc** and **ascending** so it's clearer that they're both equal?